src/constants/index.tsx

```diff
@@ -256,7 +256,7 @@ export const BLOCKED_PRICE_IMPACT_NON_EXPERT: Percent = new Percent(JSBI.BigInt(
 // used to ensure the user doesn't send so much ETH so they end up with <.01
 export const MIN_ETH: JSBI = JSBI.exponentiate(JSBI.BigInt(10), JSBI.BigInt(16)) // .01 ETH
 
-export const DEFAULT_TOKEN_LIST = 'ipfs://QmSbyVo6Kz5BuqyAHYcN7UkeCk5cALFp6QmPUN6NtPpDWL'
+export const DEFAULT_TOKEN_LIST = 'ipfs://QmbZ2F2miMNxfNjSpigmAMUpaTJSFdoPD2cBCJnB5HHFW1'
```

Comment:
A slight typo here:

name: CoW Protocol Token

![image](https://user-images.githubusercontent.com/1926216/159570365-cd5d7583-ecd7-4719-89ed-529830a5ee2b.png)

Ref in contract

![image](https://user-images.githubusercontent.com/1926216/159570855-c908874b-50e2-4748-8654-5225aa1c37bf.png)
https://etherscan.io/address/0xDEf1CA1fb7FBcDC777520aa7f396b4E015F497aB#readContract

Reply:
Fixed now with updated token name.